CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# CONTRIBUTING
+
+Contributions to the XRPL-Standards repo are free, open source, and follow the following document publishing process.
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS" (XRPL Standard) or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [MIT License](LICENSE).
+
+All contributors grant everyone:
+
+Copyright – a royalty-free license to use the copyrights for their contributions.
+Patent – a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+
+
+## Process
+
+The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+
+### Gathering Feedback Before Submitting
+
+Please gather community input before opening a PR. Collecting such feedback helps to refine your concept. This step is required.
+
+Start a [Discussion](https://github.com/XRPLF/XRPL-Standards/discussions) under this repo.
+
+The title should follow the naming convention `XLS-0000d {Title}`, where `0000` is a unique number for the XLS, `d` indicates that the document is a Draft (in progress), and `{Title}` is a descriptive title for the proposed document.
```

Comment:
How do I know what new unique number to take?

Reply:
Perhaps no number until PR?

Reply:
Since all XLS starts in discussions, it should be pretty easy to spot the largest number. In case a double happens, it can be fixed by a maintainer? I think the number is important, as they are commonly used to reference features and ideas throughout the community.